Sources/NIO/ChannelPipeline.swift

```diff
@@ -944,7 +957,141 @@ extension ChannelPipeline {
     /// - returns: A future that will be completed when all of the supplied `ChannelHandler`s were added.
     public func addHandlers(_ handlers: ChannelHandler...,
                             position: ChannelPipeline.Position = .last) -> EventLoopFuture<Void> {
-        return addHandlers(handlers, position: position)
+        return self.addHandlers(handlers, position: position)
+    }
+
+    /// Synchronously adds the provided `ChannelHandler`s to the pipeline in the order given, taking
+    /// account of the behaviour of `ChannelHandler.add(first:)`.
+    ///
+    /// - Important: Must be called on the `EventLoop`.
+    /// - Parameters:
+    ///   - handlers: The array of `ChannelHandler`s to add.
+    ///   - position: The position in the `ChannelPipeline` to add the handlers.
+    /// - Returns: A result representing whether the handlers were added or not.
+    internal func addHandlersSync(_ handlers: [ChannelHandler],
```

Comment:
Should this be `_addHandlersSync`, with the leading underscore to match `_addHandlerSync`?

Reply:
I think `_addHandlerSync` (previously `_add`, not sure why it was underscored) should be `addHandlerSync` and this should be `fileprivate func addHandlersSync`